### PR TITLE
102 release notes

### PR DESCRIPTION
**Pull request by nishitha-burman**
**Rendered page section for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-1987#10124522

Writer/Editor pass: vchapel

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -34,6 +34,44 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+The following items are now stable:
+
+* The [Server Certificate API](/microsoft-edge/webview2/reference/win32/icorewebview2_14?view=webview2-1.0.1248-prerelease&preserve-view=true) which provides an option to trust the server's TLS certificate at the application level and render the page without prompting the user about TLS or providing the ability to cancel the web request.
+
+*  The [ClearBrowsingData API](/microsoft-edge/webview2/reference/win32/icorewebview2profile2?view=webview2-1.0.1248-prerelease&preserve-view=true) which allows developers to programmatically clear specific data types for a duration:
+   * `clearBrowsingDataInTimeRange`
+   * `clearBrowsingDataAll`
+
+<!-- ====================================================================== -->
+## There is no prerelease package
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:61]:
```suggestion
There is no corresponding prerelease package.

```
Maybe move up.

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -34,6 +34,43 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+
+<!-- ====================================================================== -->
+
+## 1.0.1245.22
+  
+Release Date: June 15, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1245.22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1245.22)  
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 102.1245.22 or higher.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:46]:
```suggestion
For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 102.0.1245.22 or higher.
```

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -34,6 +34,43 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+
+<!-- ====================================================================== -->
+
+## 1.0.1245.22
+  
+Release Date: June 15, 2022
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:42]:
```suggestion
Release Date: June 14, 2022  
```

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -34,6 +34,43 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+### General
+
+#### Promotions
+
+The following items are now stable:
+
+* The [Server Certificate API](/microsoft-edge/webview2/reference/win32/icorewebview2_14?view=webview2-1.0.1245.22&preserve-view=true) which provides an option to trust the server's TLS certificate at the application level. It renders the page without prompting the user about TLS or providing the ability to cancel the web request.
+
+*  The [ClearBrowsingData API](/microsoft-edge/webview2/reference/win32/icorewebview2profile2?view=webview2-1.0.1245.22&preserve-view=true) which allows developers to programmatically clear specific data types for a duration:
+   * `clearBrowsingDataInTimeRange`
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:57]: We also have a `ClearBrowsingData` and the API name should be capitalized.

```suggestion
   * `ClearBrowsingData`
   * `ClearBrowsingDataInTimeRange`
```

**Review by peiche-jessica — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/release-notes.md:45]: @mikehoffms Can we fix the version number and release date?

File: microsoft-edge/webview2/release-notes.md (hunk 4 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/release-notes.md:59]: @mikehoffms And the API names here?

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -34,6 +34,43 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+
+<!-- ====================================================================== -->
+## 1.0.1245.22
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:39]: test of entering a comment on a merged pr @vchapel

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:45]: fixed version # 2x in https://github.com/MicrosoftDocs/edge-developer/pull/2011/files

File: microsoft-edge/webview2/release-notes.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:59]: fixed in https://github.com/MicrosoftDocs/edge-developer/pull/2011/files

**Pull request merged**